app/models/rentals_item.rb

```diff
@@ -1,11 +1,13 @@
 # frozen_string_literal: true
+
 class RentalsItem < ActiveRecord::Base
   has_paper_trail
   # use inverse of so we can create this with rental blank at first
   belongs_to :rental, inverse_of: :rentals_items
-  belongs_to :item
+  belongs_to :item, optional: true
+  # TODO: remove item_type from rentals_item.
```

Comment:
Checking on this TODO

Reply:
That's definitely not a TODO for this PR. It'll be later.